django_project/frontend/templates/map.html

```diff
@@ -62,15 +62,15 @@
 
             <div class="hidden" id="site-social" style="width: 115px;">
                 <i class="medium-size-icon mdi-content-clear" id="site-social-icon-open"></i>
-                <a href="https://twitter.com/intent/tweet?text=Share and develop critical health facility data on http://healthsites.io"
+                <a onclick="javascript:window.open(this.href, '', 'menubar=no,toolbar=no,resizable=yes,scrollbars=yes,height=600,width=600');return false;" href="https://twitter.com/intent/tweet?text=Share and develop critical health facility data on http://staging.healthsites.io/"
                    class=" withripple">
                     <i class="fa fa-twitter"></i>
                 </a>
-                <a class="facebook-hs-share withripple" href="https://www.facebook.com/sharer/sharer.php?u=http%3A//healthsites.io">
+                <a class="facebook-hs-share withripple" href="https://www.facebook.com/sharer/sharer.php?u=http://staging.healthsites.io/">
```

Comment:
We shouldn't be hard coding the url like this - it will break when we go live...